R/fragments-calculate2.R

```diff
@@ -0,0 +1,324 @@
+#' @title Calculate ions produced by fragmentation, including possible modifications
+#' 
+#' @aliases calculateFragments2, modificationPositions, cumsumFragmentMasses, character,missing-method
+#'
+#' @name calculateFragments2
+#' 
+#' @param sequence character() providing a peptide sequence.
+#' 
+#' @param type character vector of target ions; possible values: 
+#' c("a", "b", "c", "x", "y", "z"). Default is type = c("b", "y").
+#' 
+#' @param z numeric with a desired charge state; default is 1.
```

Comment:
Looking at the documentation, there's a lot of overlap with `?calculateFragments`. I'm wondering of is wouldn't be better to document both functions in the same manual page. This is possible and quite easy with roxygen. You simply reference the same `@name` tag for the two functions, and only document the `calculateFragments2()` parameters here. 

See also my more general comment.